twine-upload.sh

```diff
@@ -133,6 +134,8 @@ if [[ ${INPUT_PRINT_HASH,,} != "false" || ${INPUT_VERBOSE,,} != "false" ]] ; the
     python /app/print-hash.py ${INPUT_PACKAGES_DIR%%/}
 fi
 
+TWINE_EXTRA_ARGS="--disable-progress-bar $TWINE_EXTRA_ARGS"
```

Comment:
Let's put this right into the initial arg declaration on line 123/124 instead of adding to the var later in the process.